### PR TITLE
Update to sbt 1.3.13

### PR DESCRIPTION
**Pull request by muuki88**

### CODE REVIEW

**Comment by muuki88** [comment]
Nice way for the caching stuff: https://github.com/laserdisc-io/fs2-aws/pull/417

**Comment by muuki88** [comment]
Also check the readme update script: #1356

File: src/main/mima-filters/1.3.15.backward.excludes [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ProblemFilters.exclude[ReversedMissingMethodProblem]("com.typesafe.sbt.packager.universal.UniversalKeys.containerBuildImage")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("com.typesafe.sbt.packager.graalvmnativeimage.GraalVMNativeImageKeys.graalVMNativeImageGraalVersion")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("com.typesafe.sbt.packager.graalvmnativeimage.GraalVMNativeImageKeys.com$typesafe$sbt$packager$graalvmnativeimage$GraalVMNativeImageKeys$_setter_$graalVMNativeImageGraalVersion_=")
+# added via #1279
+ProblemFilters.exclude[ReversedMissingMethodProblem]("com.typesafe.sbt.packager.docker.DockerKeys.com$typesafe$sbt$packager$docker$DockerKeys$_setter_$dockerAutoremoveMultiStageIntermediateImages_=")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("com.typesafe.sbt.packager.docker.DockerKeys.dockerAutoremoveMultiStageIntermediateImages")
+ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sbt.packager.docker.DockerPlugin.publishLocalDocker")
+
+# added via #1319
+ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sbt.packager.universal.UniversalDeployPlugin.requires")
```
Comment (muuki88) [inline src/main/mima-filters/1.3.15.backward.excludes:20]: Sorry for pinging your directly on this one @dwijnand 

I couldn't find (or didn't proplery related) anything on the mima plugin regarding this. I bumped sbt from 1.2.8 to 1.3.13 and the mima-sbt-plugin from 0.3.0 to 0.7.0 and these error started appearing.

From my perspective it looks like the object apply methods are checked differently. And the auto plugin methods require/trigger also seem to have changed.

Is this intended? From my perspective it seems like a false positive, but I'm very unsure about this :grimacing:

Comment (dwijnand) [reply src/main/mima-filters/1.3.15.backward.excludes:20]: Yeah, I think you're right.  I spotted this the other day, but didn't spend time investigating it... https://github.com/lightbend/mima/commit/7086ffcdfcef5b0cffdcbbc3e2c6d28387bab067#diff-b2ef32f56d72f7fa17d20a86d157f6e4R39

**Pull request merged**